src/reducers/storageobject/grid/__tests__/StorageObjectReducer-test.js

```diff
@@ -34,16 +31,7 @@ const comingFromBackend = [
   }
 ]
 
-
 describe('StorageUnitReducer', () => {
-  it('Initial state is set', () => {
```

Comment:
I have created new one for LOAD_SEVERAL_SUCCESS. This was the old one. But if you want I can add it back.